backoffice/src/Controllers/ImageController.cs

```diff
@@ -0,0 +1,23 @@
+using Droits.Services;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Droits.Controllers;
+
+public class ImageController : BaseController
+{
+    private readonly IImageService _service;
+    private readonly IHttpClientFactory _httpClientFactory;
+
+    public ImageController(IImageService service, IHttpClientFactory httpClientFactory)
+    {
+        _service = service;
+        _httpClientFactory = httpClientFactory;
+    }
+    
+    public async Task<IActionResult> DisplayImage(Guid id)
+    {
+        var image = await _service.GetImageAsync(id);
```

Comment:
You'll probably want a check on this image and return not found if it doesn't exist or something (maybe even a placeholder file, but just a 404 probably best)